test/symforce_gen_codegen_test.py

```diff
@@ -91,6 +92,11 @@ def test_gen_package_codegen_python(self) -> None:
 
         config = codegen.PythonConfig()
         cam_package_codegen.generate(config=config, output_dir=output_dir)
+        if symforce.get_symbolic_api() == "symengine":
+            generate_manifold_imu_preintegration(
+                config=config,
+                output_dir=Path(output_dir) / "sym" / "factors",
+            )
```

Comment:
I think I'd rather not do this until we actually have something using these generated functions in python

Reply:
Sure. Removed.